guacamole/src/main/webapp/app/login/loginModule.js

```diff
@@ -23,4 +23,4 @@
 /**
  * The module for the login functionality.
  */
-angular.module('login', ['element']);
+angular.module('login', ['element', 'navigation']);
```

Comment:
We were going to have the login page redirect directly the the "home" page, but never got around to it.